### ISSUE TITLE
(Helm) Cannot override storage size for master / data using customvalues.yaml

### ISSUE THREAD

**Issue by jsirianni** [opendistro-for-elasticsearch/opendistro-build#163]
This may be GKE specific, I have not tried on other Kubernetes distributions.

[Helm readme](https://github.com/opendistro-for-elasticsearch/opendistro-build/tree/master/helm) says default values for storage are `gp2-encrypted` and `50Gi`
```
elasticsearch.master.storageClassName | Elasticsearch master storageClassName | "gp2-encrypted"

elasticsearch.master.storage | Elasticsearch master storage | "50Gi"
```

By default I am getting `standard` and `8Gi`. I expected the standard storage class, as that is the default for GKE. I am confused as to why 8Gi is used.

I tried setting the values like this 
```
elasticsearch:
  securityConfig:
    enabled: true
    internalUsersSecret: "internal-users-config"
  master:
    storage: "50Gi"
    storageClassName: "standard"
    replicas: 3
  client:
    replicas: 2
  data:
    replicas: 5
    storage: "100Gi"
    storageClassName: "standard"
```

My config is respected except for the storage size. Am I doing something wrong? 

```
❯ kubectl get pv | grep opendistro-vol
pvc-036fad57-8d7a-4bbf-9180-a0317aebaffa   8Gi        RWO            Delete           Bound    opendistro-vol/data-opendistro-vol-opendistro-es-master-2         standard                112s
pvc-88d6c7ca-226d-400d-9770-e516157f4957   8Gi        RWO            Delete           Bound    opendistro-vol/data-opendistro-vol-opendistro-es-master-1         standard                7m33s
pvc-8af0db99-f476-4501-9ee0-10b93b5f7868   8Gi        RWO            Delete           Bound    opendistro-vol/data-opendistro-vol-opendistro-es-data-1           standard                114s
pvc-9b2933b4-bc5e-4ed4-9848-8e3ddd4fdf8f   8Gi        RWO            Delete           Bound    opendistro-vol/data-opendistro-vol-opendistro-es-master-0         standard                7m52s
pvc-aa4abd57-0c18-42e5-a3c0-6cda0252ca2b   8Gi        RWO            Delete           Bound    opendistro-vol/data-opendistro-vol-opendistro-es-data-0           standard                7m52s
pvc-d27d0714-e245-4e36-8ba9-0bdec6ef1c2b   8Gi        RWO            Delete           Bound    opendistro-vol/data-opendistro-vol-opendistro-es-data-2           standard                89s
pvc-d46777c9-8cd4-41b4-869b-b1b908f86419   8Gi        RWO            Delete           Bound    opendistro-vol/data-opendistro-vol-opendistro-es-data-3           standard                66s
pvc-e35e7f31-0ff5-4dc3-b4fa-d3e231c23faf   8Gi        RWO            Delete           Bound    opendistro-vol/data-opendistro-vol-opendistro-es-data-4           standard                48s
```

**Comment by jsirianni** [comment]
Looks like the documentation is wrong. I found the correct config by looking at the default values file
```
  master:
    persistence:
      enabled: true
      size: "50Gi"
```

I will file a pull request to fix the doc.

**Comment by peterzhuamazon** [comment]
Hi @jsirianni ,
Thanks for the contribution and help us correct the error.
I have reviewed the PR and found a few typos.
Please feel free to resolve the discussions and I will check again tomorrow.

Thanks.

**Comment by peterzhuamazon** [comment]
This issue has been resolves with the merge of #164, thanks @jsirianni.
